cpp/purify/operators.h

```diff
@@ -435,8 +404,8 @@ init_degrid_operator_2d(const Vector<t_real> &u, const Vector<t_real> &v, const
       cellx, celly);
   auto direct = directDegrid;
   auto indirect = indirectDegrid;
-  const t_real op_norm = details::power_method<T>({direct, M, indirect, N}, power_iters, power_tol,
-                                                  T::Random(imsizex * imsizey));
+  const t_real op_norm = sopt::algorithm::power_method<T>({direct, M, indirect, N}, power_iters, power_tol,
+                                                  T::Ones(imsizex * imsizey));
```

Comment:
Did we need this to be initialized to `Ones` after all?

Reply:
It is probably good for consistency, but it is not needed.